apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Utilities.cs

```diff
@@ -115,49 +115,52 @@ bool ShouldEscape(byte b)
         }
 
         internal static async Task RetryOperationUntilCompleted(
-            Func<Task<bool>> fn,
+            Func<CallSettings, Task<bool>> fn,
```

Comment:
I'd prefer not to change too much in this PR. This method is called in two places; in one of those, that would be absolutely fine. In the other, we'd have to be fabricating an exception... which admittedly we're already doing *in this method*, but it means it's a bit more of a change.

Note that it would still need to be a `Func<CallSettings, Task>` as it needs to be asynchronous - at least in one case.